report-pox-krypton.js

```diff
@@ -310,8 +310,31 @@ const staging_db_path = 'chainstate/chain-00000080-testnet/blocks/staging.db'
 //                                            vtxindex INT NOT NULL
 //     );
 
-const data_root_path = `${root}${process.argv[3] || process.argv[2]}`
-const use_txs = process.argv[2] === '-t'
+// const data_root_path = `${root}${process.argv[3] || process.argv[2]}`
+// const use_txs = process.argv[2] === '-t'
+
+let use_txs = false;
+let use_csv = false;
+var myArgs = process.argv.slice(2);
+// console.log('myArgs: ', myArgs);
+
+for (let j = 0; j < myArgs.length; j++){
+  switch (myArgs[j]) {
+    case '-t':
+      use_txs = true;
+      break;
+    case '-csv':
+      use_csv = true;
+      break;
+    default:
+      if (myArgs[j].startsWith('/tmp')) {
```

Comment:
this is too limiting, if someone uses a path outside `/tmp`, especially if they set the node's `working_dir`.  Instead, test whether that's the last argument, which we can assume is a path.

Reply:
I did it this way to try and make the inputs more resilient, so someone could put the options in any order and the script would still run. I *could* assume the last one is the path, but what if it's not? 

What do you think of the following approaches instead?
1. detect if there is a forward (and/or backward?) slash, since that would be a path not an option
1. create an option for `-p --path` and process the argument directly following it, i.e. `-p /tmp/stacks-testnet-etc`

Reply:
forcing the options to come before the path is a good approach, so last in this case would be sufficient, and does not deviate too much from a lot of conventions

Reply:
fixed in b6f3a5c